src/main/java/com/botdarr/api/sonarr/SonarrApi.java

```diff
@@ -137,9 +137,8 @@ public List<SonarrShow> lookupContent(String search) throws Exception {
       }
 
       @Override
-      public List<SonarrShow> lookupItemById(String id) {
-        //TODO: if sonarr has a lookup by id, implement
-        return Collections.emptyList();
+      public List<SonarrShow> lookupItemById(String id) throws Exception {
+        return lookupShows("tvdb:" + id);
```

Comment:
so this function technically uses series/lookup api to lookup shows, this works (I don't have any idea, but if it works I'm fine with it)?

Reply:
Yep. Got the info from [this thread](https://forums.sonarr.tv/t/add-series-api-improvements/9317) on Sonarr forum. You can also give a look on the API itself.
https://sonarr.tv/docs/api/#/SeriesLookup/get_api_v3_series_lookup
![image](https://github.com/shayaantx/botdarr/assets/27465604/f2ea36b1-fb16-49c2-bbab-5ef041f56814)